src/core/thread/topology.hpp

```diff
@@ -639,12 +639,42 @@ class Neighbor : public InstanceLocatorInit
     LinkMetricsSeriesInfo *RemoveForwardTrackingSeriesInfo(const uint8_t &aSeriesId);
 
     /**
-     * This method removes all the Series and return the data structures to the Pool
+     * This method removes all the Series and return the data structures to the Pool.
      *
      */
     void RemoveAllForwardTrackingSeriesInfo(void);
 
-#endif
+    /**
+     * This method gets the Enh-ACK Probing metrics (this `Neighbor` object is the Probing Subject).
+     *
+     */
+    otLinkMetrics GetEnhAckProbingMetrics(void) const { return mEnhAckProbingMetrics; }
```

Comment:
Can we consider returning `const otLinkMetrics &` here? (This can help avoid creating a copy of the `otLinkMetrics` when we return it.